engine/src/main/java/org/hibernate/search/engine/search/sort/dsl/FieldSortMissingValueBehaviorStep.java

```diff
@@ -38,6 +38,30 @@
 	 */
 	N first();
 
+	/**
+	 * Give documents with missing values the highest value when sorting.
+	 * <p>
+	 * This instruction depends on whether the sort order is being ascending or descending.
+	 * <p>
+	 * The documents with missing values will be at the top in case of the descending order.
+	 * Otherwise, in case of the ascending order - documents will be at the bottom.
```

Comment:
Maybe this is more straightforward?

```suggestion
	 * This puts documents with missing values last when using ascending order,
	 * or first when using descending order.
```